index.js

```diff
@@ -37,6 +37,10 @@ var pify = module.exports = function (obj, P, opts) {
 		P = Promise;
 	}
 
+	if (typeof opts === 'string') {
+		return processFn(obj[opts], P, {}).bind(obj);
```

Comment:
I think the intention of `pify(x, 'foo')` isn't obvious, should `pify` bind `foo` to the non-promisified or the (new) promisified object – both are reasonable, at least sometimes.
```js
return processFn(obj[opts], P, {}).bind(obj);
// vs.
var objP = processFn(obj, P, {});
return objP[opts].bind(objP);
```